app/views/discovered_hosts/_discovered_hosts_list.html.erb

```diff
@@ -35,7 +35,7 @@
       <% if SETTINGS[:organizations_enabled] -%>
         <td class="hidden-tablet hidden-xs"><%= host.organization.try(:name) %></td>
       <% end -%>
-      <td class="hidden-tablet hidden-xs"><%= host.subnet %></td>
+      <td class="hidden-tablet hidden-xs"><%= host.primary_interface.subnet %></td>
```

Comment:
any reason why not to delegate ip, subnet etc to primary interface on the model ?